etc/helm/pachyderm/templates/pachd/license-secret.yaml

```diff
@@ -0,0 +1,13 @@
+{{- /*
+SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
+SPDX-License-Identifier: Apache-2.0
+*/ -}}
+{{- if and (not .Values.pachd.enterpriseLicenseKeySecretName) .Values.pachd.enterpriseLicenseKey }} 
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pachyderm-license
+  namespace: {{ .Release.Namespace }}
+stringData:
+  enterprise-license-key: {{ .Values.pachd.enterpriseLicenseKey }}
+{{- end }}
```

Comment:
I think this is a nice bootstrapping experience.  You can deploy manually, move this secret to your secrets manager, and then remove the license key from the values, and check that in safely.